tests/Functional/Schema/PostgreSQLSchemaManagerTest.php

```diff
@@ -598,6 +598,32 @@ public static function autoIncrementTypeMigrations(): iterable
             'bigint->int' => ['bigint', 'integer', 'INT'],
         ];
     }
+
+    public function testPartitionTable(): void
+    {
+        $this->connection->executeStatement('DROP TABLE IF EXISTS partitioned_table');
+        $this->connection->executeStatement(
+            'CREATE TABLE partitioned_table (id INT PRIMARY KEY) PARTITION BY LIST (id);'
+        );
+        $this->connection->executeStatement('CREATE TABLE partition PARTITION OF partitioned_table DEFAULT;');
```

Comment:
This table should now be hidden from the schema manager, did I get that right? In that case, can we assert that the table does not appear when introspecting the schema?

Reply:
yeah, I've created to assert it is hidden, I've just forgotten to do it :facepalm:

Reply:
done